routes/api/comments.js

```diff
@@ -0,0 +1,57 @@
+//2 rts
+const express = require('express');
+const {asyncHandler} = require('../utils');
+const { restoreUser } = require('../../auth');
+const createError = require('http-errors');
+const router = express.Router();
+const { CommentLike } = require("../../db/models");
+
+//
+
+//Like a comment
+router.post('/comments/:id/likes', restoreUser, asyncHandler(async(req, res) => {
+
+    const { userId, commentId } = req.body;
+
+    if (!res.locals.authenticated) {
+        return next(createError(401)); 
+    } else if (res.locals.user.id === like.userId && res.locals.authenticated ) {
```

Comment:
if `res.locals.user.id !== like.userId` we should also return a 401. Also, `like` does not yet exist. You'll need to query for the like before checking.